packages/blade/src/components/Button/IconButton/IconButton.tsx

```diff
@@ -108,6 +110,7 @@ const _IconButton: React.ForwardRefRenderFunction<BladeElementRef, IconButtonPro
       onTouchEnd={onTouchEnd}
       onTouchStart={onTouchStart}
       {...makeAnalyticsAttribute(rest)}
```

Comment:
```suggestion
```
I think we can remove this {...makeAnalyticsAttribute(rest)}, since we are now using ...rest, which handles the analytics attributes.